platform/ext/target/nordic_nrf/nrf5340/target_cfg.h

```diff
@@ -45,6 +45,23 @@ struct memory_region_limits {
  */
 int32_t spu_init_cfg(void);
 
+/**
+ * \brief Configures the Security Protection Unit (peripheral security).
+ *
+ * \return  Returns error code.
+ */
+int32_t spu_periph_init_cfg(void);
```

Comment:
Shouldn't the return value be rather of type `tfm_plat_err_t`?
Same goes for `spu_init_cfg`.

Reply:
No, I prefer to leave the imt32_t because this is how the other targets implement these functions (or better: `mpc_init_cfg()` and `ppc_init_cfg()`)

Reply:
Well, they are called from `tfm_spm_hal_init_isolation_hw()` and their return codes are anyway translated to `TFM_PLAT_ERR_*` ones. AN539 is an exception in this regard as it uses `tfm_plat_err_t` directly.
My point is that `ARM_DRIVER_OK` returned by those functions seems a bit artificial, as it is defined for CMSIS drivers and it requires inclusion of `Driver_Common.h`. Hence my comment. But I can of course live with it. Just wanted to explain what I meant.